src/pages/new-claim/new-claim-form.tsx

```diff
@@ -19,15 +20,16 @@ export type FormStatus = 'idle' | 'validation_failed' | 'submitting' | 'submitte
 function getValidationMessages(form: FormState, policy: Policy) {
   const messages: ValidationMessages<FormState> = {};
 
-  if (form.evidence.trim().length === 0) {
-    messages.evidence = 'Please fill in this field';
+  const evidenceResult = goSync(() => CID.parse(form.evidence.trim()));
```

Comment:
Judging from the fact that the validation is sync, it seems it doesn't check whether it actually points to an existing file.

Reply:
Correct. It's very unlikely that a user will accidentally enter a valid CID which doesn't exist.